### PR TITLE
Chain of Responsibility: php => c#/csharp

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1171,80 +1171,113 @@ Wikipedia says
...
+        /// </summary>
+        /// <param name="balance"></param>
+        public Account(float balance)
+        {
+            _balance = balance;
         }
-    }
-    
-    public function canPay($amount) : bool {
-        return $this->balance <= $amount;
```
Comment (ivandrofly) [inline README.md]: Probably a bug, this is supposed to return **true** only if current balance is greater than or equal to the amount required... (must be fixed a cross repositories)

Comment (phastmike) [reply README.md]: @ivandrofly : Original code is:
    public function canPay($amount): bool
    {
        return $this->balance >= $amount;
    }

the return evaluates the result of the comparison. The method will return true for any amount smaller or equal to the actual balance.

Probably a typo.

Comment (ivandrofly) [reply README.md]: Oh yeah :) noticed it while transcoding*. thanks for review

File: README.md [hunk 2]
```diff
@@ -1171,80 +1171,113 @@ Wikipedia says
...
-    public function pay(float $amountToPay) {
-        if ($this->canPay($amountToPay)) {
-            echo sprintf('Paid %s using %s' . PHP_EOL, $amount, get_called_class());
-        } else if ($this->successor) {
-            echo sprintf('Cannot pay using %s. Proceeding ..' . PHP_EOL, get_called_class());
-            $this->successor->pay($amountToPay);
-        } else {
-            throw Exception('None of the accounts have enough balance');
+```c#
+    /// <summary>
```
Comment (aloisdg) [inline README.md]: I dont think we need to add xml document. It adds cognitive load for not much profit

Comment (phastmike) [reply README.md]: I agree, code should be the most simple possible and follow only the required language conventions.

Comment (ivandrofly) [reply README.md]: Updated /cc @aloisdg | @phastmike

Comment (aloisdg) [reply README.md]: @ivandrofly I commit after you :)

**Comment by aloisdg** [comment]
https://dotnetfiddle.net/D9n3gS

**Comment by aloisdg** [comment]
Thank you @ivandrofly

**Comment by ivandrofly** [comment]
> https://dotnetfiddle.net/D9n3gS

Nice :), I was using github's default editor

**Comment by aloisdg** [comment]
You are brave 😄

**Pull request merged**